tests/test_setup.py

```diff
@@ -326,6 +327,9 @@ def test_cmake_minimum_required_version_keyword():
     "See https://github.com/conda/conda/issues/508",
 )
 @pytest.mark.skipif(not is_site_reachable("https://pypi.org/simple/cmake/"), reason="pypi.org website not reachable")
+@pytest.mark.xfail(
+    sys.platform.startswith("cygwin"), strict=False, reason="Cygwin needs a release of scikit-build first"
```

Comment:
Do you want to make this `strict=True` with a note to also update the skipped tests, or just remember to update these once a release gets up to PyPI?

Reply:
I'm okay to just remember - hopefully the release won't be too far in the future, and I'll be trying to remove xfails occasionally.